<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>ScoreboardWidget</class>
    <widget class="QWidget" name="ScoreboardWidget">
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>234</width>
                <height>93</height>
            </rect>
        </property>
        <property name="minimumSize">
            <size>
                <width>80</width>
                <height>60</height>
            </size>
        </property>
        <property name="windowTitle">
            <string>Form</string>
        </property>
        <layout class="QHBoxLayout" name="scoreboardLayout">
            <property name="spacing">
                <number>0</number>
            </property>
            <property name="leftMargin">
                <number>0</number>
            </property>
            <property name="topMargin">
                <number>0</number>
            </property>
            <property name="rightMargin">
                <number>0</number>
            </property>
            <property name="bottomMargin">
                <number>0</number>
            </property>
            <item>
                <widget class="QGroupBox" name="scoreboardGroupBox">
                    <property name="title">
                        <string>Scoreboard</string>
                    </property>
                    <property name="alignment">
                        <set>Qt::AlignCenter</set>
                    </property>
                    <property name="flat">
                        <bool>false</bool>
                    </property>
                    <property name="checkable">
                        <bool>false</bool>
                    </property>
                    <layout class="QHBoxLayout" name="scoreboardGroupBoxLayout">
                        <property name="spacing">
                            <number>0</number>
                        </property>
                        <property name="leftMargin">
                            <number>0</number>
                        </property>
                        <property name="topMargin">
                            <number>0</number>
                        </property>
                        <property name="rightMargin">
                            <number>0</number>
                        </property>
                        <property name="bottomMargin">
                            <number>0</number>
                        </property>
                        <item>
                            <layout class="QVBoxLayout" name="firstTeamLayout">
                                <item>
                                    <widget class="QLabel" name="firstTeamColor">
                                        <property name="text">
                                            <string/>
                                        </property>
                                        <property name="alignment">
                                            <set>Qt::AlignCenter</set>
                                        </property>
                                    </widget>
                                </item>
                                <item>
                                    <widget class="DynamicFontSizeQLabel" name="firstTeamScore">
                                        <property name="sizePolicy">
                                            <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
                                                <horstretch>0</horstretch>
                                                <verstretch>0</verstretch>
                                            </sizepolicy>
                                        </property>
                                        <property name="font">
                                            <font>
                                                <weight>75</weight>
                                                <bold>true</bold>
                                            </font>
                                        </property>
                                        <property name="text">
                                            <string>0</string>
                                        </property>
                                        <property name="alignment">
                                            <set>Qt::AlignCenter</set>
                                        </property>
                                    </widget>
                                </item>
                            </layout>
                        </item>
                        <item>
                            <layout class="QVBoxLayout" name="middleLayout">
                                <item>
                                    <widget class="DynamicFontSizeQLabel" name="labelX">
                                        <property name="sizePolicy">
                                            <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
                                                <horstretch>0</horstretch>
                                                <verstretch>0</verstretch>
                                            </sizepolicy>
                                        </property>
                                        <property name="font">
                                            <font>
                                                <weight>75</weight>
                                                <bold>true</bold>
                                            </font>
                                        </property>
                                        <property name="text">
                                            <string>x</string>
                                        </property>
                                        <property name="alignment">
                                            <set>Qt::AlignCenter</set>
                                        </property>
                                    </widget>
                                </item>
                            </layout>
                        </item>
                        <item>
                            <layout class="QVBoxLayout" name="secondTeamLayout">
                                <item>
                                    <widget class="QLabel" name="secondTeamColor">
                                        <property name="baseSize">
                                            <size>
                                                <width>0</width>
                                                <height>0</height>
                                            </size>
                                        </property>
                                        <property name="text">
                                            <string/>
                                        </property>
                                        <property name="alignment">
                                            <set>Qt::AlignCenter</set>
                                        </property>
                                    </widget>
                                </item>
                                <item>
                                    <widget class="DynamicFontSizeQLabel" name="secondTeamScore">
                                        <property name="sizePolicy">
                                            <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
                                                <horstretch>0</horstretch>
                                                <verstretch>0</verstretch>
                                            </sizepolicy>
                                        </property>
                                        <property name="font">
                                            <font>
                                                <weight>75</weight>
                                                <bold>true</bold>
                                            </font>
                                        </property>
                                        <property name="text">
                                            <string>0</string>
                                        </property>
                                        <property name="alignment">
                                            <set>Qt::AlignCenter</set>
                                        </property>
                                    </widget>
                                </item>
                            </layout>
                        </item>
                    </layout>
                </widget>
            </item>
        </layout>
    </widget>
    <customwidgets>
        <customwidget>
            <class>DynamicFontSizeQLabel</class>
            <extends>QLabel</extends>
            <header>soccer-common/Gui/Widgets/DynamicFontSizeQLabel/DynamicFontSizeQLabel.h</header>
        </customwidget>
    </customwidgets>
    <resources/>
    <connections/>
</ui>
